src/Uri.php

```diff
@@ -200,6 +200,10 @@ private function buildFromRawUri($rawUri)
             }
         }
 
+        if ($this->parts['scheme'] === 'file' && preg_match('/^[A-Z]:/', $this->parts['path'])) {
+            $this->parts['path'] = '/' . strtr($this->parts['path'], '\\', '/');
```

Comment:
I understand the need for the leading slash, but I don't think there should be a `strtr` at this point.